std/format/package.d

```diff
@@ -45,61 +45,95 @@ $(TR $(TH Function Name) $(TH Description)
    $(REF_ALTTEXT $(D unformatValue), unformatValue, std, format, read) are
    used for the plumbing.
 
-   Format_String:
+$(SECTION3 Format Strings)
 
-   <a name="format-string">$(I Format strings)</a>
-   consist of characters interspersed with $(I format
-   specifications). Characters are simply copied to the output (such
-   as putc) after any necessary conversion to the corresponding UTF-8
-   sequence.
+The functions of this package use $(I format strings), which describe
+the layout of an other string $(MDASH) in case of writing of the
+output string and in case of reading of the input string. A format
+string essentially consists of text interspersed with $(I format
+specifiers). A $(I format specifier) starts with a percentage sign,
+followed by some optional $(I parameters) and ends with a $(I format
+indicator) which can be either a simple $(I format character) or a
+$(I compound indicator).
 
```

Comment:
Suggested rewrite of section:

"The functions contained in this package use $(I format strings). A format string describes the layout of another string for reading or writing purposes. A format string is composed of normal text interspersed with  $(I format specifiers).
A format specifier starts with a percentage sign (`%`), followed by one or more $(I parameters) and ends with a
 $(I format indicator). A format indicator may be a simple $(I format character) or a $(I compound indicator)."

Reply:
"followed by one or more parameters": Can also be zero, like in `%s`. Is "followed by optional parameters" ok?

Reply:
I would say: "optionally followed by one or more parameters"